.github/workflows/automation.yml

```
@@ -51,14 +51,21 @@
     steps:
     - uses: actions/checkout@v4
     - name: scala-bionic
-      run: docker build . --file ./scala/Dockerfile.bionic --tag scala-bionic:$(date +%s)
-
+      run: docker build . --file ./scala/build/Dockerfile.bionic --tag "scala-bionic:$(date +%s)"
+      
   build-scala-mantic:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)